### PR TITLE
App info banner

### PR DESCRIPTION
**Pull request by bentheredonethat**
report info that shows if Apps are aligned are not.

### CODE REVIEW

**Comment by bentheredonethat** [comment]
CC @TanmayShah-xilinx

File: apps/machine/zynqmp_r5/platform_info.c [hunk 1]
```diff
@@ -182,6 +202,60 @@ int platform_init(int argc, char *argv[], void **platform)
...
+	xil_printf("### HOST IPI_DEV_NAME: %x.ipi\n", ipi_info[host][0]);
+	xil_printf("### HOST SHM_DEV_NAME: %x.shm\n", rsc_table);
+	xil_printf("### HOST RSC_MEM_PA: 0%lx\n", rsc_table);
+	xil_printf("### HOST VRING_MEM_PA: 0%lx\n", SHARED_MEM_PA);
+	xil_printf("### HOST SHARED_BUF_PA: 0%lx\n", SHARED_MEM_PA + SHARED_BUF_OFFSET);
+	xil_printf("### HOST IPI_CHN_BITMASK: 0x%x\n", ipi_info[remote][1]);
+	xil_printf("### EXPECTED IPI VALUES FOR HOST (RPMSG USERSPACE ONLY): ######\n");
+	xil_printf("### HOST IPI_DEV_NAME: %x.ipi\n", ipi_info[host][0]);
+	xil_printf("### HOST IPI REG VALUE: <0 %d 4>\n", ipi_info[host][2] - 32);
+	xil_printf("########################################\n");
```
Comment (arnopo) [inline apps/machine/zynqmp_r5/platform_info.c:258]: Is it intentional not to have the same order of traces for the host and the remote?

File: apps/machine/zynqmp_r5/platform_info.c [hunk 2]
```diff
@@ -229,6 +303,7 @@ platform_create_rpmsg_vdev(void *platform, unsigned int vdev_index,
 		goto err2;
 	}
 	xil_printf("initializing rpmsg vdev\r\n");
+	report_setup(role);
```
Comment (arnopo) [inline apps/machine/zynqmp_r5/platform_info.c:306]: Additional seems that there are 2 times in code` xil_printf("initializing rpmsg vdev\r\n");`
could you take opportunity to clean it

File: apps/machine/zynqmp_r5/platform_info.c (hunk 1 above)
Comment (bentheredonethat) [reply apps/machine/zynqmp_r5/platform_info.c:258]: I was trying to be consistent. but there is no hard requirement from my persepctive.

File: apps/machine/zynqmp_r5/platform_info.c (hunk 2 above)
Comment (bentheredonethat) [reply apps/machine/zynqmp_r5/platform_info.c:306]: sure.

File: apps/machine/zynqmp/platform_info.c [hunk 3]
```diff
@@ -80,6 +80,26 @@
 #define SHARED_BUF_SIZE     0x40000UL
 #endif /* !SHARED_BUF_SIZE */
 
+#ifdef versal /* Versal case */
```
[the remark this answers is not in the document]
Comment (bentheredonethat) [reply apps/machine/zynqmp/platform_info.c:83]: its a symbol provided in the AMD-Xilinx BSP denoting which SoC is targeted for RPU application (versal is one of the symbols used.)

[the remark this answers is not in the document]
Comment (bentheredonethat) [reply apps/machine/zynqmp/platform_info.c:83]: I'll see if I can fix the Xilinx BSP first to generate all caps symbol (VERSAL). if not will keep this as lower case.

**Review by edmooring — COMMENTED** [review]
Please address @arnopo comments.

**Comment by bentheredonethat** [comment]
closing for now.

**Pull request closed without merging**